mnefun/_mnefun.py

```diff
@@ -812,8 +855,6 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
     fname_out : str
         Output filename to use to store the resulting head positions
         on the local machine.
-    host : str
-        The SSH/scp host to run the command on.
```

Comment:
I figured if `host = p.sws_sss` or the remote identified by params this argument doesn't need to be explicit.